doc/private_net_sync.rst

```diff
@@ -0,0 +1,306 @@
+    [2019-04-02 19:59:42.520265] [0x000041f0] [info]    ReadingC:\Users\nilse\AppData\Roaming\Web3\keys\9067d973-1c8d-fa86-a312-14c90188f610.json
+    Enter a passphrase with which to secure this account:Please confirm the passphrase by entering it again: Created key 623b80dd-d008-4cd4-dd06-c36f0f64296c
+    Address: 007e13502a8b226b0f19e7412db75352dc1d0760
+
+    aleth account new
+    [2019-04-02 19:59:53.847216] [0x000011b4] [info]    ReadingC:\Users\nilse\AppData\Roaming\Web3\keys\4f04a5ed-87e4-1e4d-4367-604db42bdcff.json
+    [2019-04-02 19:59:53.849343] [0x000011b4] [info]    ReadingC:\Users\nilse\AppData\Roaming\Web3\keys\623b80dd-d008-4cd4-dd06-c36f0f64296c.json
+    [2019-04-02 19:59:53.850400] [0x000011b4] [info]    ReadingC:\Users\nilse\AppData\Roaming\Web3\keys\84258fde-b0d9-747e-b70f-f55e14831192.json
+    [2019-04-02 19:59:53.850400] [0x000011b4] [info]    ReadingC:\Users\nilse\AppData\Roaming\Web3\keys\9067d973-1c8d-fa86-a312-14c90188f610.json
+    Enter a passphrase with which to secure this account:Please confirm the passphrase by entering it again: Created key ab921356-8c9e-54ff-e3e7-da5c2f7aa685
+    Address: 002c73acd4bc217998966964d27f0ee79a47befb
+
+
+2. Add each address generated in the previous step to the ``accounts`` section of your chain configuration file (we'll refer to this as ``config.json``) along with the desired balance in wei. For example, the following initializes each account with 2 ether (2000000000000000000 wei):
+
+::
+
+    "accounts": {
+      "0000000000000000000000000000000000000001": { "precompiled": { "name": "ecrecover", "linear": { "base": 3000, "word": 0 } }, "balance": "0x01" },
+      "0000000000000000000000000000000000000002": { "precompiled": { "name": "sha256", "linear": { "base": 60, "word": 12 } }, "balance": "0x01" },
+      "0000000000000000000000000000000000000003": { "precompiled": { "name": "ripemd160", "linear": { "base": 600, "word": 120 } }, "balance": "0x01" },
+      "0000000000000000000000000000000000000004": { "precompiled": { "name": "identity", "linear": { "base": 15, "word": 3 } }, "balance": "0x01" },
+      "0000000000000000000000000000000000000005": { "precompiled": { "name": "modexp" }, "balance": "0x01" },
+      "0000000000000000000000000000000000000006": { "precompiled": { "name": "alt_bn128_G1_add", "linear": { "base": 500, "word": 0 } }, "balance": "0x01" },
+      "0000000000000000000000000000000000000007": { "precompiled": { "name": "alt_bn128_G1_mul", "linear": { "base": 40000, "word": 0 } }, "balance": "0x01" },
+      "0000000000000000000000000000000000000008": { "precompiled": { "name": "alt_bn128_pairing_product" }, "balance": "0x01" },
+      "007e13502a8b226b0f19e7412db75352dc1d0760": {
+          "balance" : "0x2000000000000000000"
+      },
+      "002c73acd4bc217998966964d27f0ee79a47befb": {
+          "balance" : "0x2000000000000000000"
```

Comment:
I think it's not 2 ether, it's 0x2000000000000000000 wei (hex). Should be `"balance" : "2000000000000000000"`

Reply:
You are correct, typo on my part